service/src/main/kotlin/fi/espoo/evaka/pis/EmployeeQueries.kt

```diff
@@ -535,6 +535,7 @@ fun Database.Read.getInactiveEmployees(now: HelsinkiDateTime): List<EmployeeId>
         SELECT max(ts)
         FROM unnest(ARRAY[e.last_login, d.updated, dg.updated]) ts
     ) < ${bind(now)} - interval '56 days'
+    AND e.active = true
```

Comment:
maybe function name could be updated, e.g. `getEmployeesToDeactivate`, and could be changed to `private` too. or maybe even deleted and the query moved inside only use place `deactivateInactiveEmployees`